website/docs/r/express_route_gateway.html.markdown

```diff
@@ -0,0 +1,76 @@
+---
+subcategory: "Network"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_express_route_gateway"
+description: |-
+  Manages an ExpressRoute gateway.
+---
+
+# azurerm_express_route_gateway
+
+Manages an ExpressRoute gateway.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "example-resources"
+  location = "West US"
```

Comment:
_minor_ we're using West Europe for all of the (newer) examples can we update this:

```suggestion
  location = "West Europe"
```